www/header.inc

```diff
@@ -190,6 +190,9 @@ if( !strcasecmp('Test Result', $tab) && !@$nosubheader && !defined('EMBED') )
                           if ($check == 'lighthouse') {
                             $lighthouse = $urlGenerator->resultPage("lighthouse");
                             echo "<li class=\"$check\"><a href=\"$lighthouse\"><h2 class=\"{$grade['class']}\">{$grade['grade']}</h2></a>{$grade['description']}</li>";
+                          }else if ($check == 'security') {
+                            $snykUrl = "https://snyk.io/test/website-scanner/?test={$id}";

```

Comment:
Should the snyk url be able to take a full URI for a test result so it can also be used by private instances (like SpeedCurve)?  Otherwise the checks should be behind a check for the $private instance global (in common.inc) and only enabled for the public instance.

Reply:
I think at the moment probably better to land this integration first as only available to the public WPT website and I'll re-visit this once we have deployed it as I also need to make internal adjustments on the snyk side to load any other URL if it's not the public instance.

So I can update my latest commit for GetSetting (https://github.com/WPO-Foundation/webpagetest/commit/c162b28cc212bcadae63cae1469470f132a51192) to also ensure we're on the public instance like this: 

```
if (GetSetting('securityInsights) && !$privateInstall) {
```

or is the GetSetting() enough to make sure that this is anyway not turned on by default for private instances?

Reply:
GetSetting would be enough. If someone turns the setting on then they're just shooting themselves in the foot (or collecting the grade but without the link to the UI).

Reply:
Ok thanks!